README.rst

```diff
@@ -116,17 +68,24 @@ Make sure you have installed the following dependencies:
   * libpcap-dev
   * libssl-dev
   * libffi-dev
+  * libgmp-dev
+  * openssl
+  * libyaml-dev
```

Comment:
This is also not required, since the pyyaml will work also without the yaml dev headers.